cartography/models/aws/ec2/volumes.py

```diff
@@ -61,3 +77,32 @@ class EBSVolumeSchema(CartographyNodeSchema):
             EBSVolumeToEC2Instance(),
         ],
     )
+
+
+@dataclass(frozen=True)
+class EBSVolumeInstanceProperties(CartographyNodeProperties):
```

Comment:
Looking at the sample `DESCRIBE_INSTANCES`, the only properties about Volumes that are returned are just `volumeid` and `deleteontermination`, right?
https://github.com/lyft/cartography/blob/facb63bcbac6b68eec0fd2ea3f6b0550ac40eb10/tests/data/aws/ec2/instances.py#L390-L395

Reply:
Good point, will update